src/fragments/lib/logging/ios/sending_logs/10_sending_logs.mdx

```diff
@@ -0,0 +1,26 @@
+```swift
+let logger = Amplify.Logging.logger(forCategory: "Storage", forNamespace: "<your-code-namespace>")
+let message = """
+	{
+		"action": "value,
+		"subAction": {
+			"completed": true
+		}
+	}
+"""
+logger.error(message)
+```
+
+The following are existing Amplify category names:
```

Comment:
The following are Amplify category names that are used by default by Amplify when automatically logging errors from the library.